Kickstarter-iOS/Views/HeaderFooterViewProtocol.swift

```diff
@@ -1,5 +1,5 @@
 import Foundation
 
 protocol HeaderFooterViewProtocol {
-  func configure(with text: String)
+  func configure(with text: String?)
```

Comment:
So that we can pass `nil` and hide the label (in addition to kidding the wrapper) ... this should be proper behaviour.

Hiding = label with no text